src/styles/theme/defaultTheme.ts

```diff
@@ -336,5 +375,61 @@ export const defaultTheme = createTheme({
         noSsr: true,
       },
     },
+    MuiInputBase: {
+      styleOverrides: {
+        root: {
+          padding: 0,
+        },
+        input: {
+          padding: '0 !important',
+          backgroundColor: COLORS.white,
+          color: COLORS.grayDark,
+          '&::placeholder': {
+            color: 'rgba(49,67,90,0.7)',
```

Comment:
let's use hex `#647080` and add it to COLORS

Reply:
Added the opacity to focus as a solution.

Reply:
Actually that is not a solution. Will modify.